src/main/java/webapp/openapi.json

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
it is good that we can include swagger-UI, can you check whether can we use YAML instead of JSON?

Reply:
yes, YAML file can also be launched.

Reply:
YAML is definitely the most widely used of the 2 in this context.